python.d/nvidia_smi.chart.py

```diff
@@ -0,0 +1,260 @@
+# -*- coding: utf-8 -*-
+# Description: nvidia-smi netdata python.d module
+# Author: Steven Noonan (tycho)
+
+import copy
+import subprocess
+import threading
+import xml.etree.ElementTree as et
+import time
+from bases.FrameworkServices.SimpleService import SimpleService
+from bases.collection import find_binary
+
+CHART_TEMPLATES = {
+    'pci_bandwidth': {
+        'options': [None, 'PCI express bandwidth utilization', 'KB/s', 'nvidia_smi', 'nvidia_smi', 'area'],
+        '_metrics': { 'pci': [ 'rx_util', 'tx_util' ] },
```

Comment:
You've got some formatting issues here and a couple of places below too.

This line should look like this:

```
'_metrics': {'pci': ['rx_util', 'tx_util']},
```

For future reference, you can easily check for stuff like this by using [flake8](https://pypi.org/project/flake8/).  The only difference from the default behavior is that we use a line length limit of 120 characters instead of 79 (so you can get behavior identical to what our checking does by just passing `--max-line-length=120` to flake8).

Reply:
Yes I appreciate formatting, I have recently switched to vscode and dont have all my PEP8 stuff setup yet.